src/calendarmanager.cpp

```diff
@@ -224,12 +224,13 @@ QString NemoCalendarManager::getNotebookColor(const QString &notebookUid) const
 
 void NemoCalendarManager::cancelAgendaRefresh(NemoCalendarAgendaModel *model)
 {
-    mAgendaRefreshList.removeOne(model);
+    if (!mAgendaRefreshList.isEmpty())
+        mAgendaRefreshList.removeOne(model);
```

Comment:
That particular reference is talking about things like .at(), and so on. removeOne is essentially "a for loop plus a return once it's removed the item", so it's safe without.